src/appleseed.python/bindproject.cpp

```diff
@@ -305,7 +305,9 @@ void bind_project()
     bpy::enum_<ProjectFileReader::Options>("ProjectFileReaderOptions")
         .value("Defaults", ProjectFileReader::Defaults)
         .value("OmitReadingMeshFiles", ProjectFileReader::OmitReadingMeshFiles)
-        .value("OmitProjectFileUpdate", ProjectFileReader::OmitProjectFileUpdate);
+        .value("OmitProjectFileUpdate", ProjectFileReader::OmitProjectFileUpdate)
+        .value("OmitSearchPaths", ProjectFileReader::OmitSearchPaths)
```

Comment:
Intuitively, without reading the rest of the code, I'm not sure what `OmitSearchPaths` means or achieves. Maybe the name could be made more descriptive.

Edit: Now that I've read the code, maybe `OmitAddingSearchPaths` would be clearer?